ui/src/app/views/concept-homepage/component.ts

```diff
@@ -69,13 +70,15 @@ export class ConceptHomepageComponent implements OnInit {
 
   // Maps Domain Name with number of concept selected
   selectedConceptDomainMap: Map<String, number>  = new Map<string, number>();
-  concepts: Array<Concept> = [];
+  concepts: Array<ConceptInfo> = [];
   conceptsCache: Array<ConceptCacheSet> = [];
   completedDomainSearches: Array<Domain> = [];
   placeholderValue = '';
   vocabularies: Array<VocabularyCountSelected> = [];
   wsNamespace: string;
   wsId: string;
+  selectConceptMap = new Map<number, ConceptInfo>();
+  conceptsToAdd: Concept[];
```

Comment:
It would be better to dynamically compute this than risk the state going out of sync. If that's too much effort and we're still planning to move off clarity data grid, fine to leave for now.